### DISCUSSION TITLE
Action runner (ubuntu) includes 'flex' but not 'libfl2' and 'libfl-dev' => missing 'FlexLexer.h'

### DISCUSSION THREAD

**Discussion by vypxl** [community/community#45029] [Actions]
### Select Topic Area

Bug

### Body

The ubuntu action runner has the [`flex`](https://github.com/westes/flex) tool preinstalled, but not the associated development packages `libfl2` and `libfl-dev`. This means that when compiling a C or C++ application using `flex`, users like me will run into the issue that the header file required to use lexers generated by `flex`, namely `FlexLexer.h` is not available.

The two packages are recommended to be installed alongside `flex`, so I think it should be a sane default to include them.

**Comment by tlegen-k** [comment]
Have same issue with `flex` when trying to use it for compilation.

**Reply by AranVink** [reply]
Same for me, I specifically ask it to do `apt install flex`, tells me it's already installed, but I'm missing libfl.so.2/libfl.so, so in turn `ar` fails to run for me.
